ansible/roles/ci/code_style_check/license/files/gather_missing_licences.py

```diff
@@ -71,12 +71,13 @@ def get_changes_in_pr(data):
             file_path = os.path.join(data.path, file_changed)
             _, extension = os.path.splitext(file_path)
             if not extension:  # We sometimes have python files without extensions
-                with open(file_path, 'r') as code_file:
-                    firstline = code_file.readline().strip()
-                    if "#!/usr/bin/env python" in firstline:  # Can use to tell if py file
-                        payload = (file_path, "py")
-                        if payload not in data.changed_files:
-                            data.changed_files.append(payload)
+                if os.path.exists(file_path) and os.path.isfile(file_path):
+                    with open(file_path, 'r') as code_file:
+                        firstline = code_file.readline().strip()
+                        if "#!/usr/bin/env python" in firstline:  # Can use to tell if py file
```

Comment:
I think not all of our shebangs are having this version yet. Can you add a note / another accepted version?

Reply:
It's true that some of our files still have #!/usr/bin/python, but we agreed that we want to converge tot the one mentioned here, so it might be good to report them as non valid header somehow?

Reply:
What is the other accepted version but I can add that :)

Reply:
I think that would be a seperate story to have a script to go through and list all the repos that have it like this and change it